src/pykeen/models/unimodal/trans_r.py

```diff
@@ -157,7 +157,7 @@ def interaction_function(
         # evaluate score function, shape: (b, e)
         return -linalg.vector_norm(h_bot + r - t_bot, dim=-1) ** 2
 
-    def score_hrt(self, hrt_batch: torch.LongTensor) -> torch.FloatTensor:  # noqa: D102
+    def score_hrt(self, hrt_batch: torch.LongTensor, **kwargs) -> torch.FloatTensor:  # noqa: D102
```

Comment:
`mode != None` would result in silent incorrect behaviour, as it would return transductive scores in any case?

Reply:
At the moment it doesn’t do any checking that an invalid one gets passed. Maybe we can add those checks in the training loop and evaluator but doing a check in every single models seems like a huge amount of code and potentially adding unnecessary conditionals to runtime

Reply:
We didn't include in this PR the logic of get_entity_representations that will actually use this mode arg, but it's going to be in the main PR, so it seems ok for now

Reply:
> We didn't include in this PR the logic of get_entity_representations that will actually use this mode arg, but it's going to be in the main PR, so it seems ok for now

this will only fix it for new-style models, right?

Reply:
yeah... not really sure how to deal with the rest at the moment (perhaps more motivation to upgrade all old-style models)!